src/plugins/intellisense/latex-parsing.tsx

```diff
@@ -204,26 +215,12 @@ export function getPartialFunctionCall(
       const oldCursor = cursor;
       cursor = cursor.parent?.parent?.[-1];
 
-      // TODO: maybe add support for cursed operatornames in the future?
-      // or more likely builtins
-      const ltx = cursor?.latex?.();
-      const ltx2 = cursor?.[-1]?.latex?.();
+      const ltx = rawTryGetMathquillIdent(cursor)?.ident;
       if (ltx && isIdentStr(ltx) && cursor?.[1]?.ctrlSeq === "\\left(") {
         return {
           ident: latexStringToIdentifierString(ltx) as string,
           paramIndex,
         };
-      } else if (
-        ltx2 &&
-        ltx &&
-        isIdentStr(ltx2 + ltx) &&
-        cursor?.[1]?.ctrlSeq === "\\left("
-      ) {
-        const str = latexStringToIdentifierString(ltx2 + ltx);
-        return {
-          ident: str as string,
-          paramIndex,
-        };
```

Comment:
This was handling the case where you need to look at two mq nodes to get a full identifier. I think that's only subscripts like `f_{123}(x)`, but the `rawTryGetMathquillIdent` correctly does that.